charts/pokt-portal-v2/templates/secret.yaml

```diff
@@ -1,7 +1,8 @@
+# TODO do we need this?
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ .Release.Name }}-vars
+  name: {{ .Release.Name }}-secret-vars
```

Comment:
What is the thought process behind this secret ?

Reply:
I think we should remove it, see https://pokt.height.app/portal-team/T-19189?task=T-19198